osu!stream/Support/Android/GameBaseAndroid.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.IO;
+using Android.App;
+using osum.AssetManager;
+using osum.Audio;
+using osum.GameModes;
+using osum.GameModes.SongSelect;
+using osum.Input;
+using osum.Input.Sources;
+using Xamarin.Essentials;
+
+namespace osum.Support.Android
+{
+    internal class GameBaseAndroid : GameBase
+    {
+        private readonly Activity activity;
+
+        public static bool IsInitialized;
+        public GameWindowAndroid Window;
+
+        public GameBaseAndroid(Activity activity, OsuMode mode = OsuMode.Unknown) : base(mode)
+        {
+            this.activity = activity;
+
+            NativeAssetManagerAndroid.Manager = activity.Assets;
+        }
+
+        public override void Run()
+        {
+            // Before we run anything, let's check if this is the first run...
+            // If this is the first run, let's import the packaged beatmaps from our assets!
+            // This is unfortunately required because Android sucks, thanks Google.
+            if (Config.GetValue("firstrun", true))
```

Comment:
Not 100% sure on this, but I guess we'll go with it for now. Will revisit before potentially releasing to the play store.